doc/whats_new/v0.21.rst

```diff
@@ -100,6 +107,10 @@ Changelog
 - |Fix| Fixed bug in :func:`tree.export_text` when the tree has one feature and 
   a single feature name is passed in. :pr:`14053` by `Thomas Fan`.
 
+- |Fix| Fixed an issue with :func:`plot_tree` where it display
+  entropy calculations even for `gini` criterion in DecisionTreeClassifiers.
+  :pr:`13947` by :user:`Frank Hoang <fhoang7>`.
```

Comment:
This was not part of 0.21.2 as far as I can tell.

Reply:
I agree